samples/kofu-servlet-validation/src/main/kotlin/com/sample/Handlers.kt

```diff
@@ -0,0 +1,15 @@
+package com.sample
+
+import org.springframework.web.servlet.function.ServerRequest
+import org.springframework.web.servlet.function.ServerResponse.badRequest
+import org.springframework.web.servlet.function.ServerResponse.ok
+
+@Suppress("UNUSED_PARAMETER")
+class UserHandler {
+
+    fun createApi(request: ServerRequest) =
+            request.body(User::class.java)
```

Comment:
This line should be `request.body<User>()` after `org.springframework.web.servlet.function.ServerResponseExtensions` is available in spring-framework.

Reply:
You mean `ServerRequestExtensions` right? If so, `request.body<User>()` is expected to be available, see https://github.com/spring-projects/spring-framework/blob/master/spring-webmvc/src/main/kotlin/org/springframework/web/servlet/function/ServerRequestExtensions.kt#L40. Any issue in using it?

Reply:
Ah, I missed it